docs/pages/Configuration.md

```diff
@@ -702,6 +703,17 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 3.0
+**Breaking changes:**
+
+`@ui5/builder/processors/bundlers/moduleBundler` `ModuleBundleOptions` has been modified:
+
+- `ModuleBundleOptions.debugMode` has been removed
```

Comment:
`ModuleBundleOptions` refers to the JSDoc name, but here we should describe the changes in the ui5.yaml configuration.